scripts/sync_sunbeam.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Clone the repository with sparse checkout
+git clone --sparse --single-branch --depth=1 https://opendev.org/openstack/sunbeam-charms.git
+cd sunbeam-charms
+
+# Set sparse checkout paths
+git sparse-checkout set charms/openstack-exporter-k8s/src/grafana_dashboards charms/openstack-exporter-k8s/src/prometheus_alert_rules
+
+# Copy the necessary files to the src directory
+cp -r charms/openstack-exporter-k8s/src/grafana_dashboards ../src/
+cp -r charms/openstack-exporter-k8s/src/prometheus_alert_rules ../src/
+
+# Clean up
+cd ..
+rm -rf sunbeam-charms
```

Comment:
eeeeehhh `rm -rf`s give me shivers. If you **really** want to make the script cleanup after itself for the times when we may want to run it locally (because if it's run as part of the CI then the whole runner would be eventually torn down) I'd prefer using a temporary dir with a unique name and absolute paths.

See this link for a good example using traps: https://stackoverflow.com/a/34676160 (I'd just avoid the `-p` option and create the directory under `$TMPDIR`)